packages/colorpickers/src/styled/common/StyledRange.ts

```diff
@@ -60,24 +60,51 @@ const trackLowerStyles = (styles: string, modifier = '') => {
   `;
 };
 
-const colorStyles = (props: IStyledRangeProps & ThemeProps<DefaultTheme>) => {
-  const thumbBackgroundColor = getColorV8('neutralHue', 100, props.theme);
-  const thumbBorderColor = thumbBackgroundColor;
-  const thumbActiveBackgroundColor = getColorV8('neutralHue', 200, props.theme);
-  const thumbActiveBorderColor = getColorV8('primaryHue', 600, props.theme);
-  const thumbFocusBorderColor = getColorV8('primaryHue', 400, props.theme);
-  const thumbHoverBackgroundColor = thumbActiveBackgroundColor;
-  const thumbHoverBorderColor = thumbHoverBackgroundColor;
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const thumbBackgroundColor = getColor({
+    theme,
+    variable: 'background.default'
+  });
+  const thumbBorderColor = getColor({
+    theme,
+    variable: 'border.default',
+    dark: { offset: -100 },
+    light: { offset: 100 }
+  });
```

Comment:
[nit] now that the thumbs have a visible border, can we check that the `transition` animation is updated to include the border?

Reply:
Transition animations appear to be inherited from `Range`:

https://github.com/zendeskgarden/react-components/blob/e1bac19bbdbf99ec6a8131ce6a12c1db6e9888b4/packages/forms/src/styled/range/StyledRangeInput.ts#L142-L144

![Screenshot 2024-07-24 at 6 31 22 AM](https://github.com/user-attachments/assets/f9394d1d-3090-48af-8846-2d8399786def)

Reply:
Ok, my bad. I wasn't viewing the `:hover` state.